lib/internal/errors.js

```diff
@@ -1329,6 +1329,9 @@ E('ERR_INVALID_RETURN_VALUE', (input, name, value) => {
   return `Expected ${input} to be returned from the "${name}"` +
          ` function but got ${type}.`;
 }, TypeError, RangeError);
+E('ERR_INVALID_SHORT_OPTION', 'options.%s.short must be a single' +
```

Comment:
Since this is only used in one place, it might not make sense to introduce a new error code. You can already do:

```
> new errors.codes.ERR_INVALID_ARG_VALUE('options.foo.short', '--bar', 'must be a single character');
TypeError [ERR_INVALID_ARG_VALUE]: The property 'options.foo.short' must be a single character. Received '--bar'
```

Reply:
done.